### DISCUSSION TITLE
How to useRuntimeConfig of Nuxt3

### DISCUSSION THREAD

**Discussion by takimotoSE** [storybookjs/storybook#21759] [Help]
Hi!
I am developing with Nuxt3 and Storybook and use environment variables in `useRuntimeConfig` in Nuxt3.
However, `useRuntimeConfig` does not work in Storybook and I get the following error.
```
Cannot read properties of undefined (reading '$config')
```
Please let me know if you know anything about it.

**Version**
`nuxt`: 3.0.0
`@storybook/vue3`: 6.5.16
`@sentry/vite-plugin`: 0.2.3
`@storybook/builder-vite`: 0.4.2

**Comment by chakAs3** [comment]
hi @takimotoSE  can  you upgrade your storybook to v7 ? so i can help you in this case, for now it is really hard to go back to 6.5. and maintain it

**Comment by takimotoSE** [comment]
@chakAs3 
OK, I'll update to Storybook v7 and give it a try.
I'll call you if I have the same problem. Thank you.
